src/mono/mono/mini/mini-llvm.c

```diff
@@ -3922,6 +3927,13 @@ emit_entry_bb (EmitContext *ctx, LLVMBuilderRef builder)
 			emit_args_to_vtype (ctx, builder, ainfo->type, ctx->addresses [reg]->value, ainfo, args);
 			break;
 		}
+		case LLVMArgVtypeInSIMDReg: {
+			LLVMValueRef arg = LLVMGetParam (ctx->lmethod, pindex);
+
+			ctx->addresses [reg] = build_alloca_address (ctx, ainfo->type);
+			LLVMBuildStore (builder, arg, convert (ctx, ctx->addresses [reg]->value, LLVMPointerType (LLVMTypeOf (arg), 0)));
+			break;
```

Comment:
To be llvm 14.x compatible, this should use build_ptr_cast (builder, ctx->addresses [reg]->value, pointer_type (LLVMTypeOf (arg)))